CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ All notable changes to this project will be documented in this file.
     * #### Removed
 * ### `nidcpower` (NI-DCPower)
     * #### Added
+        * Pass Python interpreter information to driver runtimes that support it. This is used by NI in order to better understand client usage.
```

Comment:
This is True but @marcoskirsch do you think there's a better way to explain it?

Reply:
Rephrased it.